src/components/permit/MainContainer.tsx

```diff
@@ -58,30 +59,35 @@ const MainContainer = ({
             className=" z-50 focus-visible:outline-blue-600 focus-visible:outline-none font-mono text-blue-600 bg-input-bg w-full py-3 px-3 rounded-md mb-4 uppercase"
             type={'text'}
           />
-        </form>
-        <div className="flex items-center mt-8">
-          <Button
-            data-testid="permit-button"
-            text={isCreatePermit ? 'Create' : 'Revoke'}
-            classes={{ container: 'mr-3' }}
-            isDisabled={!inputData}
-            onClick={() =>
-              isRevokePermit ? handleRevokePermit() : handleCreatePermit()
-            }
-          />
           <Link
             href={
               isRevokePermit
                 ? '/applicant/permit?type=create'
                 : '/applicant/permit?type=revoke'
             }
           >
-            <a className="z-50">
+            <a className="z-50 underline">
               {isCreatePermit
                 ? 'Revoke a query permit'
                 : 'Create a query permit'}
             </a>
           </Link>
+        </form>
+        <div className="flex justify-between items-center mt-8">
+          <Button
+            text="Back"
+            style={BUTTON_STYLES.OUTLINE}
+            onClick={() => router.push('/applicant')}
+          />
+          <Button
+            data-testid="permit-button"
+            text={isCreatePermit ? 'Create' : 'Revoke'}
+            classes={{ container: 'mr-3' }}
+            isDisabled={!inputData}
+            onClick={() =>
+              isRevokePermit ? handleRevokePermit() : handleCreatePermit()
+            }
+          />
```

Comment:
We have a component for both buttons in one - I think it's called NavigationButtons. Can you use it?